ui/app/AppLayouts/Chat/popups/community/CreateChannelPopup.qml

```diff
@@ -115,7 +115,7 @@ StatusDialog {
                     }
                 }
                 input.asset.color: colorDialog.color.toString()
-                rightPadding: 6
+                leftPadding: 16
```

Comment:
Here we use the default rightPadding but Figma says the leftPadding should be 16 here in the popup

Reply:
16 is the default padding. Can use Theme.padding instead of the hardcoded paddings